client/components/jetpack/upsell-product-wpcom-plan-card/index.tsx

```diff
@@ -0,0 +1,200 @@
+			context: 'The Plan name, ie- Starter, Explorer, Creator,.',
+		},
+	} );
+
+	const upsellImageAlt = translate( 'Get %(productName)s plan', {
+		args: {
+			productName: plan.getTitle(),
+			context: 'The Plan name, ie- Starter, Explorer, Creator,.',
+		},
+		textOnly: true,
+	} );
+
+	const upsellImageUrl = useMemo( () => {
+		if ( isJetpackBackupSlug( nonManageProductSlug ) ) {
+			return BackupImage;
+		}
+		if ( isJetpackScanSlug( nonManageProductSlug ) ) {
+			return ScanImage;
+		}
+		if ( isJetpackSearchSlug( nonManageProductSlug ) ) {
+			return SearchImage;
+		}
+		return DefaultImage;
+	}, [ nonManageProductSlug ] );
+
+	const { displayName, description } = item;
+
+	const renderPriceDetails = () => {
+		return (
+			<>
+				<div className="display-price__details" aria-hidden="true">
+					<span className="display-price__billing-time-frame">
+						{ translate( 'per month, %(rawPrice)s billed annually, excl. taxes', {
+							args: {
+								rawPrice: ! isFetchingPrices
+									? formatCurrency( originalPrice ?? 0, currencyCode ?? '', {
+											stripZeros: true,
+											isSmallestUnit: true,
+									  } )
+									: '',
+							},
+							comment: 'Excl. Taxes is short for excluding taxes',
+						} ) }
+					</span>
+				</div>
+			</>
+		);
+	};
+
+	const renderProductCardBody = () => {
+		return (
+			<>
+				<div className="upsell-product-card__price-plan">Creator plan</div>
+				<div className="upsell-product-card__price-container">
+					<DisplayPrice
+						isFree={ ! isFetchingPrices && originalPrice === 0 }
+						currencyCode={ currencyCode }
+						originalPrice={ displayPrice ?? 0 }
+						pricesAreFetching={ isFetchingPrices }
+						billingTerm={ billingTerm }
+						productName={ displayName }
+						hideSavingLabel={ false }
+						customTimeFrameBillingTerms={ renderPriceDetails() }
+					/>
+				</div>
+				<div className="upsell-product-card__included" aria-hidden="true">
+					{ translate( 'Included with %(planName)s plan:', {
+						args: {
+							planName: plan.getTitle(),
+						},
+					} ) }
+				</div>
+				<ul className="upsell-product-card__features is-wpcom-features">
+					{ jetpackFeaturesObject &&
+						jetpackFeaturesObject.map( ( feature, i ) => (
+							<li className="upsell-product-card__features-item" key={ i }>
+								<>
+									<Gridicon size={ 18 } icon="checkmark" /> { feature.getTitle() }
+								</>
+							</li>
+						) ) }
+				</ul>
+			</>
+		);
+	};
+
+	return (
+		<>
+			<JetpackRnaActionCard
+				headerText={ displayName }
+				subHeaderText={ description }
+				onCtaButtonClick={ onCtaButtonClick }
+				ctaButtonURL={ ctaButtonURL }
+				ctaButtonLabel={ ctaButtonLabel }
+				cardImage={ upsellImageUrl }
+				cardImageAlt={ upsellImageAlt }
+			>
+				{ renderProductCardBody() }
+			</JetpackRnaActionCard>
+		</>
+	);
+};
+
+export default UpsellProductWpcomPlanCard;
+
+export const UpsellProductWPcomPlanCardPlaceholder: React.FC = () => {
```

Comment:
I'm a bit leaning toward moving this component to a separate file if it's intended to be used across files, or remove the `export` in front of it as currently this seems only to be used in this file.

Reply:
Thanks for taking a look. 
I noticed we are not using this Placeholder in any place, we are using [this one](https://github.com/Automattic/wp-calypso/blob/b6790349c90533a8bebe1b90e81f89c81a32fcef/client/components/jetpack/upsell-product-card/index.tsx#L262) instead.
I will remove it since the other placeholder looks almost the same and works great for this case.